src/main/resources/assets/mekanism/lang/zh_CN.lang

```diff
@@ -779,68 +806,69 @@ tooltip.BoilerCasing=一种耐压方块,用于制作锅炉结构的外壳
 tooltip.BoilerValve=用于锅炉结构的阀门,能让水进入锅炉并导出产生的蒸汽
 tooltip.SecurityDesk=一个中央控制系统,用于管理你所有机器的安全设置
 
-
-
-
-
-tooltip.EnrichmentChamber=一台能将矿石磨成两份粉末的机器, !n也可以进行一些其他操作.
-tooltip.OsmiumCompressor=一台非常高级的设备,能够将锇矿石压缩成 !n多种粉末以制造相应金属锭.
+tooltip.EnrichmentChamber=一台能将矿石磨成两份粉末的机器, 也可以进行一些其他操作.
+tooltip.OsmiumCompressor=一台非常高级的设备,能够将锇矿石压缩成 多种粉末以制造相应金属锭.
 tooltip.Combiner=A machine 一台用来将粉末与圆石融合为相应矿石的机器.
-tooltip.Crusher=一台用来将金属锭磨成相应粉末的机器, !n也可以进行一些其他操作.
-tooltip.DigitalMiner=一台高度先进,以过滤为基础功能的自动采矿机, !n它可以挖掉指定的32格范围(最大)内的所有方块.
-tooltip.MetallurgicInfuser=一台用来将多种材料的金属熔炼成!n合金或其他化合物的设备.
-tooltip.PurificationChamber=一台能够将矿石加工成三块矿石疙瘩的高级设备,!n能够提供300%的矿石加工效率.
+tooltip.Crusher=一台用来将金属锭磨成相应粉末的机器, 也可以进行一些其他操作.
+tooltip.DigitalMiner=一台高度先进,以过滤为基础功能的自动采矿机, 它可以挖掉指定的32格范围(最大)内的所有方块.
+tooltip.MetallurgicInfuser=一台用来将多种材料的金属熔炼成合金或其他化合物的设备.
+tooltip.PurificationChamber=一台能够将矿石加工成三块矿石疙瘩的高级设备,能够提供300%的矿石加工效率.
 tooltip.EnergizedSmelter=一台简单的消耗通用机器能量的熔炉.
 tooltip.Teleporter=一台能够将玩家传送到其他传送器标定的各种位置的设备.
 tooltip.ElectricPump=一台能支持升级,过滤,自动弹射的高级泵
-tooltip.ElectricChest=便携的54格容量的箱子,!n具有耗能的密码锁功能.
+tooltip.ElectricChest=便携的54格容量的箱子,具有耗能的密码锁功能.
 tooltip.Chargepad=通用充能板,能够为各种模组中的物品补充能量.
-tooltip.LogisticalSorter=一台以过滤为基础功能的高级整理机器, !n能够自动将指定物品弹射到依附的物品栏或物流管道中.
-tooltip.RotaryCondensentrator=一台能够将气体和其液态形式,!n相互转化的机器.
-tooltip.ChemicalInjectionChamber=一台能够将!n矿物处理成4个碎片的精英机器, 是400%量产矿物的!n第一步.
-tooltip.ElectrolyticSeparator=一台使用!n电解将特定气体 !n分离成两种不同气体的机器.
-tooltip.PrecisionSawmill=一台更有效地处理原木和!n木制品的机器, 同时!n也能获得锯屑.
-tooltip.ChemicalDissolutionChamber=一台能够通过化学手段溶解!n矿石里的所有杂质, !n从而得到未经处理的矿物浆液的终极机器.
-tooltip.ChemicalWasher=一台能够清理未经处理的!n浆液,为结晶作准备的终极机器.
-tooltip.ChemicalCrystallizer=一台用于结晶!n净化过的矿物浆液的终极机器.
-tooltip.ChemicalOxidizer=一台能够将!n固态材料氧化成气态产物的机器.
-tooltip.ChemicalInfuser=一台能够通过注入其它两种!n气体产生新的气体的机器.
-tooltip.SeismicVibrator=一台能够利用地震震动!n提供关于这个世界!n地质结构的机器.
-tooltip.PressurizedReactionChamber=一台能够处理!n固体,液体和气体混合物,并且能!n产生气态和固态产物的高级机器.
-tooltip.PortableTank=一个便利的携带式水槽,能够让你!n无论何处都能携带!n14桶液体. 同时可以当做水桶来用!
-tooltip.FluidicPlenisher=一台能够用液体填充沟壑!n从而产生一整个湖的机器.
-tooltip.Laser=一种高级的能量传输形式!n,利用平行光线实现能量!n的直线传输.
-tooltip.LaserAmplifier=一个能用于合并,转向和!n增幅激光的方块,易于控!n制.
-tooltip.LaserTractorBeam=一个用于合并和转向激!n光的方块.会收集它所破!n坏方块的掉落物.
-tooltip.SolarNeutronActivator=一个将太阳的中子辐射!n引入自身腔体的机器,可!n缓慢制造各种同位素.
-tooltip.Oredictionificator=一个使用矿典统一并翻译!n各种物品和方块的机器.
+tooltip.LogisticalSorter=一台以过滤为基础功能的高级整理机器, 能够自动将指定物品弹射到依附的物品栏或物流管道中.
+tooltip.RotaryCondensentrator=一台能够将气体和其液态形式,相互转化的机器.
+tooltip.ChemicalInjectionChamber=一台能够将矿物处理成4个碎片的精英机器, 是400%量产矿物的第一步.
```

Comment:
The same situation also exists in en_US file.

    tooltip.ChemicalInjectionChamber=An elite machine capable of processing ores into four shards,    serving as the initial stage of 400% ore processing.